packages/host/tests/acceptance/basic-test.ts

```diff
@@ -130,6 +130,9 @@ module('Acceptance | basic tests', function (hooks) {
     realm = await TestRealm.createWithAdapter(adapter, loader, this.owner, {
       isAcceptanceTest: true,
     });
+
+    loader.registerURLHandler(realm.maybeHandle.bind(realm));
```

Comment:
I'm seeing these locally at type errors because registerURLHandler takes two arguments. I don't know why CI didn't error.

Reply:
Maybe the pnpm/git hard linking thing because this PR changes it to [one argument](https://github.com/cardstack/boxel/pull/467/files#diff-af066918a5fef32d05c1b66dc00c15a20c9593cf0b1e30e77f4fa7f459f6d3e2R149) 😯